Resources/config/swiftmailer.xml

```diff
@@ -47,9 +47,7 @@
 
         <service id="swiftmailer.transport.spool.abstract" class="Swift_Transport_SpoolTransport" public="false" abstract="true" />
 
-        <service id="swiftmailer.spool.file.abstract" class="Swift_FileSpool" public="false" abstract="true">
-            <argument>%kernel.root_dir%/../data/swiftmailer/spool</argument>
```

Comment:
we usually prefer using replaceArgument - which means we should use `<argument />` to reserve the position